src/openbook/database.py

```diff
@@ -0,0 +1,26 @@
+from collections.abc import Generator
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+from openbook.models.orm import Base
+
+DATABASE_URL = "sqlite:///mydb.db"
+
+engine = create_engine(DATABASE_URL, connect_args={"check_same_thread": False})
+
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+def get_db() -> Generator[Session, None, None]:
+    """Dependency Function."""
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
```

Comment:
Enable WAL mode. Not tested

```py
from sqlalchemy.engine import Engine
from sqlalchemy import event

@event.listens_for(Engine, "connect")
def set_sqlite_pragma(dbapi_connection, connection_record):
    cursor = dbapi_connection.cursor()
    cursor.execute("PRAGMA journal_mode=WAL;")
    cursor.close()
```

https://www.sqlite.org/wal.html
https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#foreign-key-support